src/scenes/home/signup/signup.js

```diff
@@ -139,7 +131,7 @@ class SignUp extends Component {
           {this.state.error ? <ul className={styles.errorList}>There was an error joining Operation Code:
             <li className={styles.errorMessage}>{this.state.error}</li>
           </ul> : null }
-          {this.state.success && <Redirect to="/thanks" />}
```

Comment:
Is `/thanks` used anywhere else? If not we can delete the entire component instead of just the route.